cpp/open3d/t/io/file_format/FilePLY.cpp

```diff
@@ -289,23 +281,19 @@ bool ReadPointCloudFromPLY(const std::string &filename,
 
 static e_ply_type GetPlyType(const core::Dtype &dtype) {
     if (dtype == core::UInt8) {
-        return PLY_UINT8;
+        return PLY_UCHAR;
     } else if (dtype == core::UInt16) {
         return PLY_UINT16;
     } else if (dtype == core::Int32) {
-        return PLY_INT32;
-    } else if (dtype == core::Float32) {
-        return PLY_FLOAT32;
-    } else if (dtype == core::Float64) {
-        return PLY_FLOAT64;
-    } else if (dtype == core::UInt8) {
-        return PLY_UCHAR;
-    } else if (dtype == core::Int32) {
-        return PLY_INT32;
+        return PLY_INT;
     } else if (dtype == core::Float32) {
         return PLY_FLOAT;
-    } else {
+    } else if (dtype == core::Float64) {
         return PLY_DOUBLE;
+    } else {
+        utility::LogError(
+                "Data-type {} is not supported in WritePointCloudToPLY.",
```

Comment:
Let's give more info. Tell the user the available options to convert to.

Reply:
Done.